cpp/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ set (CMAKE_FIND_NO_INSTALL_PREFIX TRUE FORCE)
 
 cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
```

Comment:
By any chance, do we have a plan to upgrade to CMake 3.15 or higher?

Reply:
Yes, to 3.17: #2194

Reply:
Great!